src/Makefile

```diff
@@ -1182,6 +1182,9 @@ KMAPSOURCE = ../runtime/keymap
 # Where to look for print resource files
 PRINTSOURCE = ../runtime/print
 
+# Where to look translated README and LICENSE files
+TRNRMLICSOURCE = ../lang
```

Comment:
a bit obscure variable name? How about `TRANSSOURCE` instead?

Reply:
Yeah, that's probably a better name.